test/handler.test.ts

```diff
@@ -116,7 +116,7 @@ describe('handlePullRequest', () => {
       return {
         addAssignees: true,
         addReviewers: false,
-        assignees: ['assignee1', 'assignee2'],
+        assignees: ['assignee1'], // , 'assignee2'],
```

Comment:
Thanks for working on this. Just one to fix.

Remove unused `assignee2` element.